scripts/zones/Valkurm_Dunes/mobs/Metal_Shears.lua

```diff
@@ -0,0 +1,42 @@
+-----------------------------------
+-- Area: Valkurm Dunes
+--  MOB: Metal Shears
+-----------------------------------
+
+-----------------------------------
+-- onMobInitialize
+-----------------------------------
+
+function onMobInitialize(mob)
+    mob:setMobMod(MOBMOD_ADD_EFFECT,mob:getShortID());
+end;
+
+-----------------------------------
+-- onAdditionalEffect Action
+-----------------------------------
+function onAdditionalEffect(mob,target,damage)
+    -- Guesstimating 1 in 3 chance to poison on melee.
+    if ((math.random(1,100) >= 33) or (target:hasStatusEffect(EFFECT_POISON) == true)) then
+        return 0,0,0;
+    else
+        local duration = math.random(10,25);
+        target:addStatusEffect(EFFECT_POISON,15,3,duration);
+        return SUBEFFECT_POISON,0,EFFECT_POISON;
```

Comment:
that zero is the message, does this mob's add effect do its poisoning silently? if so this is correct, if not you can find the message ID in status.lua :)